### PR TITLE
Loading component

### PR DESCRIPTION
**Pull request by vacas5**
Added a component for a spinner, and am showing that now when the
character list is actively fetching.

### CODE REVIEW

**Comment by vacas5** [comment]
Fixes #15

File: src/Loader.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import oval from './oval.svg';
+import './Loader.css';
+import classNames from 'classnames';
+
+class Loader extends Component {
```
Comment (nicrocs) [inline src/Loader.js]: I think this is a good candidate for a stateless component: https://facebook.github.io/react/docs/reusable-components.html#stateless-functions

You can manage the hide show based on the props, or maybe just in the container view, I'll leave a note there.

File: src/CharacterList.js [hunk 2]
```diff
@@ -55,6 +57,7 @@ class CharacterList extends Component {
         <ul className="nav_items">
           {alphabet.map((letter, index) => <AlphaLinks letter={letter} key={index} />)}
         </ul>
+        <Loader hide={this.state.characters.length} />
```
Comment (nicrocs) [inline src/CharacterList.js]: You could also hide and show this component by wrapping it in a conditional: ```{this.state.characters.length > 0 && 
           <Loader />
}```

**Comment by vacas5** [comment]
word, I will try this

**Comment by nicrocs** [comment]
👍  nicely done

**Pull request merged**